src/tweak.cpp

```diff
@@ -75,7 +75,20 @@ UniValue gettweak(const UniValue &params, bool fHelp)
 
     UniValue ret(UniValue::VOBJ);
     bool help = false;
-    for (unsigned int i = 0; i < params.size(); i++)
+    unsigned int psize = params.size();
+
+    if (psize == 0) // No arguments should return all tweaks
+    {
+        for (CTweakMap::iterator item = tweaks.begin(); item != tweaks.end(); ++item)
+        {
+            if (help)
```

Comment:
`help` is always `false` here, we can just remove the the if then and make the else condition always executed.